repose-aggregator/components/filters/filter-bundle/pom.xml

```diff
@@ -204,6 +204,11 @@
             <artifactId>cors</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.openrepose</groupId>
+            <artifactId>ip-classification</artifactId>
```

Comment:
I looked it up it was whatever-user filter. The only one currently in that paradigm is rackspace-auth-user for identity.